config/views.view.files.yml

```diff
@@ -189,8 +189,8 @@ display:
           label: Name
           exclude: false
           alter:
-            alter_text: false
-            text: ''
+            alter_text: true
+            text: "{{ filename }}<br />{{ uri }}"
```

Comment:
I think `uri` needs to be added as a hidden field before this:
![image](https://user-images.githubusercontent.com/67453/118533869-e1473280-b748-11eb-8892-9f64bbf25a89.png)

(I deployed security updates over what was on staging so I lost that as a reference.)

Reply:
It *is* hidden as a field before that :-)

Reply:
That's odd - on my local, I didn't see it as part of the view or the output, but when I added the field - as in the screenshot - it gave the output I was expecting.

Not sure what I missed but happy to approve this PR.